README.md

```diff
@@ -89,17 +89,255 @@ Name of the application pool.
+
+#### `managed_runtime_loader`
+
+Specifies the managed loader to use for pre-loading the the application pool.Note: This property was added in IIS 7.5.The default value is `webengine4.dll`
 
 #### `managed_runtime_version`
 
-The managedruntimeversion of the ApplicationPool.
+Specifies the .NET Framework version that is used by the application pool.
+
+#### `pass_anonymous_token`
+
+When true, the Windows Process Activation Service (WAS) creates and passes a token for the built-in IUSR anonymous user account to the Anonymous authentication module. The Anonymous authentication module uses the token to impersonate the built-in account. When false, the token will not be passed.Note: The IUSR anonymous user account replaces the IIS_MachineName anonymous account. The IUSR account can be used by IIS or other applications. It does not have any privileges assigned to it during setup. Valid options `true` or `false`
+
+#### `start_mode`
+
+Specifies the startup type for the application pool. Valid options `:OnDemand` or `:AlwaysRunning`
+
+#### `queue_length`
+
+Indicates to HTTP.sys how many requests to queue for an application pool before rejecting future requests. When the value set for this property is exceeded, IIS rejects subsequent requests with a 503 error. If the `load_balancer_capabilities` setting is true, the connection is closed instead of rejecting requests with a 503. For more information about `load_balancer_capabilities`, see https://www.iis.net/configreference/system.applicationhost/applicationPools/add/failure". Valid options 11 to 65535.
+
+
+#### `cpu_action`
+
+Configures the action that IIS takes when a worker process exceeds its configured CPU limit. Valid options `:NoAction`, `:KillW3wp`, `:Throttle` or `:ThrottleUnderLoad`.
+
+#### `cpu_limit`
+
+Configures the maximum percentage of CPU time (in 1/1000ths of one percent) that the worker processes in an application pool are allowed to consume over a period of time as indicated by the `cpu_reset_interval` property. If the limit set by the limit property is exceeded, an event is written to the event log and an optional set of events can be triggered. These optional events are determined by the action property.
+
+#### `cpu_reset_interval`
+
+Specifies the reset period (in minutes) for CPU monitoring and throttling limits on an application pool. When the number of minutes elapsed since the last process accounting reset equals the number specified by this property, IIS resets the CPU timers for both the logging and limit intervals.
+
+Important: The `cpu_reset_interval` value must be greater than the time between logging operations, otherwise IIS will reset counters before logging has occurred, and process accounting will not occur.
+
+Note: Because process accounting in IIS uses Windows job objects to monitor CPU times for the whole process, process accounting will only log and throttle applications that are isolated in a separate process from IIS
```

Comment:
Unless I'm missing something, I think your change doesn't make sense.  It could be case that that sentence should be fragmented.